src/stratis_cli/_actions/_top.py

```diff
@@ -72,6 +73,13 @@ def _add_update_key(proxy, key_desc, capture_key, *, keyfile_path):
 
     if capture_key:  # pragma: no cover
         password = getpass(prompt="Enter key data followed by the return key: ")
+        verify = getpass(prompt="Verify key data entered: "
+
+        try:
+            assert password == verify
+        except AssertionError as err:
+            raise StratisCliPassphraseMismatchError from err
```

Comment:
@mulkieran I'm curious what your feelings are on this but personally I'd rather just have something like:

```python
if password != verify:
    raise StratisCliPassphraseMismatchError()
```

Reply:
@jbaublitz Yes. I prefer the if-statement, also.